pycqed/analysis_v2/base_analysis.py

```diff
@@ -199,9 +204,39 @@ def process_data(self):
     def prepare_plots(self):
         pass
 
-    def run_fitting(self):
+    def prepare_fitting(self):
         pass
 
+    def run_fitting(self):
+        # This function does the fitting and saving of the parameters
+        # based on the fit_dict options.
+        # Only model fitting is implemented yet. Minimizing fitting should
+        # be implemented here.
+        fit_guess_fn = self.fit_dict['fit_guess_fn']
+        fit_fn = self.fit_dict['fit_fn']
+        fit_yvals = self.fit_dict['fit_yvals']
+        fit_xvals = self.fit_dict['fit_xvals']
+        model = lmfit.Model(fit_fn)
+        if self.do_timestamp_blocks:
```

Comment:
I still don't understand how do timestamp blocks works but I guess that will come.